src/domains/issue/controller/issue.controller.ts

```diff
@@ -4,14 +4,27 @@ import HttpStatus from 'http-status';
 import { type IssueService, IssueServiceImpl } from '@domains/issue/service';
 import { type IssueRepository, IssueRepositoryImpl } from '@domains/issue/repository';
 import { type EventRepository, EventRepositoryImpl } from '@domains/event/repository';
-import { IssueWorkedTimeParamsDTO, IssuePauseParams, type WorkedTimeDTO } from '@domains/issue/dto';
+import { IssueWorkedTimeParamsDTO, IssuePauseParams, type WorkedTimeDTO, UserProjectParamsDTO, type IssueViewDTO, DevOptionalIssueFiltersDTO } from '@domains/issue/dto';
+import { type UserRepository, UserRepositoryImpl } from '@domains/user';
+import { type ProjectRepository, ProjectRepositoryImpl } from '@domains/project/repository';
 require('express-async-errors');
 
 export const issueRouter = Router();
 
 const issueRepo: IssueRepository = new IssueRepositoryImpl(db);
 const eventRepo: EventRepository = new EventRepositoryImpl(db);
-const issueService: IssueService = new IssueServiceImpl(issueRepo, eventRepo);
+const userRepo: UserRepository = new UserRepositoryImpl(db);
+const projectRepo: ProjectRepository = new ProjectRepositoryImpl(db);
+const issueService: IssueService = new IssueServiceImpl(issueRepo, eventRepo, userRepo, projectRepo);
+
+issueRouter.post('/dev/:userId/project/:projectId', validateRequest(UserProjectParamsDTO, 'params'), validateRequest(DevOptionalIssueFiltersDTO, 'body'), async (req: Request<UserProjectParamsDTO, any, DevOptionalIssueFiltersDTO>, res: Response) => {
```

Comment:
I had problems with the validation when I sent the object as a query param. Let's find a library for future enpoints to have  al the properties inside the object parsed and then it can go through the validation middleware.